code/mission/missionparse.cpp

```diff
@@ -2934,9 +2934,12 @@ int parse_object(mission *pm, int  /*flag*/, p_object *p_objp)
 		stuff_string(temp, F_NAME, NAME_LENGTH);
 		p_objp->team_color_setting = temp;
 
-		if (Team_Colors.find(p_objp->team_color_setting) == Team_Colors.end()) {
-			mprintf(("Invalid team color specified in mission file for ship %s, resetting to default\n", p_objp->name));
-			p_objp->team_color_setting = sip->default_team_name;
+		//If team color is not default then verify
+		if (stricmp(p_objp->team_color_setting.c_str(), sip->default_team_name.c_str()) != 0) {
```

Comment:
Just a quick thing here.  SCP_strings (std::string) can just use == instead of having to use stricmp.

Reply:
Done

Reply:
Actually hang on. I do want stricmp here because we want case insensitive comparison.

Reply:
ok

Reply:
`SCP_string_lcase_equal_to` could be used